api/db/run-migrations.sh

```diff
@@ -22,17 +23,11 @@ trap finish EXIT
 envsubst < "$(dirname "${BASH_SOURCE}")/create_db.sql" > $CREATE_DB_FILE
 
 function run_mysql() {
-  if [ -f /.dockerenv ]; then
-    mysql $@
-  else
-    echo "Outside docker: invoking mysql via docker for portability"
-    docker run --rm --network host --entrypoint '' -i \
-      mysql:5.7.27 \
-      mysql $@
-  fi
+  mysql $@
 }
 
 echo "Creating database if it does not exist..."
+
 cat "${CREATE_DB_FILE}" | run_mysql -h "${DB_HOST}" --port "${DB_PORT}" -u root -p"${MYSQL_ROOT_PASSWORD}"
```

Comment:
should this be `HEX_DB_PASSWORD` ?

Reply:
HEX_DB_PASSWORD was used in mariadb, for mysql, HEX_DB_PASSWORD is not needed